src/vma/main.cpp

```diff
@@ -109,6 +109,8 @@ const char *vma_version_str = "VMA_VERSION: " PACKAGE_VERSION "-" STR(VMA_LIBRAR
 #endif
 			      ;	// End of vma_version_str - used in "$ strings libvma.so | grep VMA_VERSION"
 
+#define VMA_GIT_TITLE		"Git: "
```

Comment:
I see enough symbol places to provide more clear description. It is not critical for me. Just nice to have.